bedrock/firefox/templates/firefox/includes/download-list.html

```diff
@@ -14,4 +14,5 @@
   {%- endfor %}
 </ul>
 
-<p><a href="{{ firefox_url('desktop', 'all', 'release') }}">{{_('Available Languages') }}</a></p>
+{# The <p> below should be removed when l10n tag 'firefox_new_install_options' is removed #}
```

Comment:
Instead of using CSS & l10n tag trickery to conditionally show this link, there might be a simpler option. It looks like the current template has both of these strings already translated:

- `Firefox for other platforms`
- `Available Languages`

Could we use these in the fallback instead, removing the link from inside the modal altogether and display two links one above the other? Might that be simpler?